packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,7 +111,22 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
-const askApplicationExtensibiltyQuestions = (appExtensions) => {
+const APP_EXTENSIBILITY_TYPESCRIPT_MINIMAL_DEV_DIR = 'dev'
+
+const INITIAL_QUESTION = [
+    {
+        name: 'project.type',
+        message: 'What type of project do you want to create?',
+        type: 'list',
+        choices: [
+            {name: 'PWA Kit Project', value: 'PWAKitProject'},
+            {name: 'Application Extension', value: 'appExtensionProject'}
+        ],
+        default: 'PWAKitProject'
+    }
```

Comment:
A couple comments here. 

1. The answers kind of imply that the `Application Extension` is not pwa-kit because we prefix one answer with it, but not the other. Can you throw around a couple more options on how to word this.
2. The values are strings.. do we do this to be friendly with using options in the command line?

Reply:
1. Adjusted. Please let me know your thoughts.
2. Yes, the idea is to make it easier to pass arguments via CLI, define presets, and also help to make the branching logic more readable.